tracer/src/Datadog.Trace/AppSec/Waf/ReturnTypes.Managed/DiagnosticResultUtils.cs

```diff
@@ -0,0 +1,69 @@
+﻿// <copyright file="DiagnosticResultUtils.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.Vendors.Serilog;
+
+namespace Datadog.Trace.AppSec.Waf.ReturnTypes.Managed;
+
+#pragma warning disable SA1402 // File may only contain a single type
+
+internal static class DiagnosticResultUtils
+{
+    internal static ReportedDiagnostics ExtractReportedDiagnostics(Obj diagObject, bool noRuleDiagnoticsIsError)
```

Comment:
noRuleDiagnoticsIsError, little spelling error.

Reply:
Good, catch but I'm going to merge as is, to avoid a another CI run just to fix a typo. I'm planning to do some more work here soon, so I'll fix it then.